src/components/DocumentationTopic/TopicLinkBlockIcon.vue

```diff
@@ -9,19 +9,10 @@
 -->
 
 <template>
-  <IconOverrideProvider
-    :imageOverride="imageOverride"
-    #default="{ shouldUseOverride, themeId, iconUrl }"
-  >
-    <div class="topic-icon-wrapper" v-if="icon || shouldUseOverride">
-      <SVGIcon
-        v-if="shouldUseOverride"
-        v-bind="{ themeId, iconUrl }"
-        class="topic-icon"
-      />
-      <component v-else :is="icon" class="topic-icon" />
-    </div>
-  </IconOverrideProvider>
+  <div class="topic-icon-wrapper" v-if="imageOverride || icon">
+    <OverridableAsset v-if="imageOverride" :imageOverride="imageOverride" class="topic-icon" />
```

Comment:
I'm noticing a potential issue when testing this specifically for external SVGs or SVGs without an ID—it looks like the icon is somehow getting sized based on its viewbox, which means that it will always have the same width wherever it is used on the page.

This is a problem for the documentation intro element where we show a bigger version of the same icon that is sized much smaller in the sidebar, for example.

SVG with ID:
<img width="1436" alt="Screenshot 2022-10-05 at 4 08 00 PM" src="https://user-images.githubusercontent.com/212918/194180141-d06ad1fa-5247-48b8-bcc7-f44f46d2b59e.png">

SVG without ID:
<img width="1436" alt="Screenshot 2022-10-05 at 4 07 28 PM" src="https://user-images.githubusercontent.com/212918/194180163-d7023895-683b-49bd-a53b-ad251c3d4e86.png">

I think we should try to see if there's a way to fix that with these changes if possible. (The issues with colors still stand but they can at least _partially_ be mitigated by providing dark mode versions)

Reply:
Hey Marcus, I saw what you mean. It was happening because we did not set a width/height to the `picture` element of the Asset. Should be good now.

Reply:
I'm not sure that it's also needed on the `<picture>` element in addition to the `<img>`.